sessions/MachineLearning-100/101/101.002/1.md

```diff
@@ -0,0 +1,36 @@
+# Data exploration
+To help you to work out what data would be useful for your model, you need to find some data that has some relationship
+to the thing you want to predict. 
+
+## Types of data
+Nominal - Labels, not quantities. Distances between values aren't meaningful. Can't be ordered. Examples: Colours, names. 
+
+Ordinal - Have an order, but distances between values aren't meaningful. Example: Film rating stars. 
+
+Interval - Ordered, and distances are meaningful, but without an absolute scale. Example: Temperature in celsius.
+
+Ratio - Like interval data, but with an absolute scale. Example: Number of adults living at an address.
+
+## Features
+A "feature" is information that can be used by a machine learning model to generate predictions.
+The meaning differs depending on the type of data.
+
+- A "feature" in tabular data usually refers to the data in a column.
+- A "feature" in image data is a fragment of an image that appears many times in the dataset.
+- A "feature" in audio is a pattern in the audio signal that appears many times in the dataset.
+
+## Visualisation is your friend
+Plot the relationships between variables to see if you can immediately uncover patterns in the data.
+
+Check for anomalies in the data that could impact your models. Are there a lot of outliers?
+
+Apply filters and recodes based on what you learn from the first two steps.
+
+Cycle: Collect -> Analyse -> Clean
+
+## Correlation vs causation 
+Be careful not to jump to conclusions about correlated data. Just because two values are correlated, it 
+doesn't mean that they are related, they just have a symmetric relationship.
+
+For example, in young school children their star sign correlates with their IQ, but this correlation weakens
```

Comment:
Is this a good example? The star sign is an indicator of age within each year and that is the cause of the variation in performance. Would a better example be that the height of a school child is correlated with academic ability, but it's not the cause: they are both causally linked to age.